### PR TITLE
ContinuousWaveletTransform code and test 

### PR DESCRIPTION
**Pull request by dharak029**
ContinuousWaveletTransform .java has been refined and some initial test cases has been written.  Currently it's reading data from eic.txt file for ContinuousWaveletTransform . But, I'll add code to get  mz values, retention time and intensity from mzml file.

### CODE REVIEW

**Comment by tomas-pluskal** [comment]
Thanks, Dharak. It seems the tests are failing because some test files are missing (see the Travis test build log). 
Can you add the missing files?

**Comment by dharak029** [comment]
Hello Dr. Pluskal,

I've already added the files in my fork repository. I don't know why it's failing. The fourth commit which is failing contains output files of test case.  Should we add it or not? It can be generated by running test case.

Regards,
Dharak Shah

**Comment by tomas-pluskal** [comment]
This is from the output log:
````
-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running io.github.msdk.featdet.ADAP3D.common.algorithms.ContinuousWaveletTransformTest
Tests run: 2, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 0.06 sec <<< FAILURE! - in io.github.msdk.featdet.ADAP3D.common.algorithms.ContinuousWaveletTransformTest
testMZMLFile(io.github.msdk.featdet.ADAP3D.common.algorithms.ContinuousWaveletTransformTest)  Time elapsed: 0.009 sec  <<< FAILURE!
java.lang.AssertionError: Cannot read test data
	at io.github.msdk.featdet.ADAP3D.common.algorithms.ContinuousWaveletTransformTest.testMZMLFile(ContinuousWaveletTransformTest.java:95)
testEICFile(io.github.msdk.featdet.ADAP3D.common.algorithms.ContinuousWaveletTransformTest)  Time elapsed: 0.001 sec  <<< FAILURE!
java.lang.AssertionError: Cannot read test data
	at io.github.msdk.featdet.ADAP3D.common.algorithms.ContinuousWaveletTransformTest.testEICFile(ContinuousWaveletTransformTest.java:45)
Results :
Failed tests: 
  ContinuousWaveletTransformTest.testEICFile:45 Cannot read test data
  ContinuousWaveletTransformTest.testMZMLFile:95 Cannot read test data
````
So check the line numbers 45 and 95 and see why it is failing there

**Comment by dharak029** [comment]
From the output log it looks like it's not able to find the files. I've given the path of the files as below.
private static final String TEST_DATA_PATH = "..\\msdk-featdet-ADAP-3D\\src\\test\\resources\\". should I change it to private static final String TEST_DATA_PATH = "src/test/resources/"  as other test cases of msdk? in my eclipse it was giving me errors.

File: msdk-featdet/msdk-featdet-ADAP-3D/src/test/java/io/github/msdk/featdet/ADAP3D/common/algorithms/ContinuousWaveletTransformTest.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+import io.github.msdk.featdet.ADAP3D.datamodel.Result;
+import io.github.msdk.datamodel.rawdata.RawDataFile;
+import io.github.msdk.io.mzml.MzMLFileImportMethod;
+import io.github.msdk.datamodel.rawdata.ChromatographyInfo;
+
+import io.github.msdk.datamodel.rawdata.MsScan;
+
+public class ContinuousWaveletTransformTest {
+	
+	private static final String TEST_DATA_PATH = "..\\msdk-featdet-ADAP-3D\\src\\test\\resources\\";
```
Comment (tomas-pluskal) [inline msdk-featdet/msdk-featdet-ADAP-3D/src/test/java/io/github/msdk/featdet/ADAP3D/common/algorithms/ContinuousWaveletTransformTest.java]: I think here is the problem. Use 
````
TEST_DATA_PATH = "src/test/resources/";
````

**Comment by tomas-pluskal** [comment]
What errors do you get in Eclipse?

**Comment by dharak029** [comment]
Currently I did not get any errors. But, previously when I was writing the test at that time it was not able to find the file with this value of test_data_path. I've changed the value of the variable. Please see if it's running or not. Thanks Dr. Pluskal.

**Comment by tomas-pluskal** [comment]
Looks good, thanks!

**Comment by dharak029** [comment]
Thanks Dr. Pluskal.

**Pull request merged**